components/forms/PasswordReset/index.jsx

```diff
@@ -0,0 +1,38 @@
+export const passwordResetFormTestIds = {
+  form: 'password-reset-form',
+  emailLabel: 'email',
+  emailField: 'input-email',
+  passwordResetButton: 'btn-password-reset',
+};
+
+const PasswordResetForm = () => {
+  return (
+    <form className="w-full" data-test-id={passwordResetFormTestIds.form}>
+      <div className="mb-6">
+        <label
+          className="block text-white text-base-small font-extrabold mb-2"
+          htmlFor="email"
+          data-test-id={passwordResetFormTestIds.emailLabel}
+        >
+          Email
+        </label>
+        <input
+          className="w-full h-14 bg-white/20 text-white text-base-large rounded-xl px-3 py-1.5 focus:outline-none focus:shadow-none"
+          id="email"
+          data-test-id={passwordResetFormTestIds.emailField}
+          type="email"
+          required
+        />
+      </div>
+      <button
+        className="w-full h-14 bg-white text-base-large font-extrabold rounded-xl"
+        data-test-id={passwordResetFormTestIds.passwordResetButton}
+        type="submit"
+      >
+        Send Recovery Email
```

Comment:
It seems that there is a minor alignment issue with the button text:

| Figma | Current render |
|---|---|
| ![image](https://user-images.githubusercontent.com/77609814/160742068-0d3b8bbf-296e-4861-aa2f-f03cd76a029a.png)|![image](https://user-images.githubusercontent.com/77609814/160742127-9a973cbf-6da9-4920-b794-dd619871906d.png)|

(It is quite hard to see I should admint 🙇 )
![image](https://user-images.githubusercontent.com/77609814/160742530-9b288726-2136-41e5-976d-c0d1d8a63117.png)

Reply:
done in 62702641a4f14b18d1a77bfdc6c11f87694fc841